pontoon/base/templates/translate.html

```diff
@@ -144,22 +144,68 @@
             <li class="horizontal-separator">By Translation Status</li>
-            <li class="all" data-type="all"><span class="status fa"></span>All</li>
-            <li class="missing" data-type="missing"><span class="status fa"></span>Missing</li>
-            <li class="fuzzy" data-type="fuzzy"><span class="status fa"></span>Fuzzy</li>
-            <li class="suggested" data-type="suggested"><span class="status fa"></span>Suggested</li>
-            <li class="translated" data-type="translated"><span class="status fa"></span>Translated</li>
+
+            <li class="all" data-type="all">
+              <span class="status fa"></span>
+              <span class="title">All</span>
+              <span class="count"></span>
+            </li>
+            <li class="missing" data-type="missing">
+              <span class="status fa"></span>
+              <span class="title">Missing</span>
+              <span class="count"></span>
+            </li>
+            <li class="fuzzy" data-type="fuzzy">
+              <span class="status fa"></span>
+              <span class="title">Fuzzy</span>
+              <span class="count"></span>
+            </li>
+            <li class="suggested" data-type="suggested">
+              <span class="status fa"></span>
+              <span class="title">Suggested</span>
+              <span class="count"></span>
+            </li>
+            <li class="translated" data-type="translated">
+              <span class="status fa"></span>
+              <span class="title">Translated</span>
+              <span class="count"></span>
+            </li>
+
+            <li class="horizontal-separator">By Smart Criteria</li>
+
+            <li class="untranslated" data-type="untranslated">
+              <span class="status fa"></span>
+              <span class="title">Untranslated</span>
+            </li>
+            <li class="unchanged" data-type="unchanged">
+              <span class="status fa"></span>
+              <span class="title">Unchanged</span>
+            </li>
+            <li class="has-suggestions" data-type="has-suggestions">
+              <span class="status fa"></span>
+              <span class="title">Has Suggestions</span>
+            </li>
+
+            <li class="horizontal-separator">By Translation Time Range</li>
+
+            <li class="time-range clearfix" data-type="time-range">
+              <span class="status fa"></span>
+              <label for="from">From<input type="datetime" id="from" disabled></label>
+              <label for="to">To<input type="datetime" id="to" disabled></label>
+            </li>
+            <li class="time-range-toolbar">
+              <button class="shortcut" data-hours="1">Last Hour</button> &middot;
+              <button class="shortcut" data-hours="24">Day</button> &middot;
+              <button class="shortcut" data-hours="168">Week</button> &middot;
+              <button class="shortcut" data-hours="720">Month</button>
```

Comment:
Yeah, I know this pain behind this type of widgets -> "month" sounds less confusing for the user but requires more work. Then "30 days" is less "human" but explains everything perfectly.